docs/virtual_machines/istio_service_mesh.md

```diff
@@ -0,0 +1,147 @@
+# Istio service mesh
```

Comment:
Looking at the render https://deploy-preview-466--kubevirt-user-guide.netlify.app/virtual_machines/istio_service_mesh/#istio-service-mesh. Istio is now available through search, but not listed in the menu as far as I can tell. Would you add it there via https://github.com/kubevirt/user-guide/blob/ef98ff5d7231e78fa3aaac3120242b0f032a5681/docs/virtual_machines/.pages